README.md

```diff
@@ -151,20 +139,56 @@ ansible_ssh_private_key_file=conf/towerperf_id_rsa
 ^C
 ```
 
+Destroy IBM Cloud Instances
+----------------------------
+
+Run the `ansible-playbook` command with following `EXTRA` cmdline arguments
+
+    ansible-playbook cleanup.yml -e ibmcloud_vsi_count=2 \
+                                 -e ibmcloud_vpc_name_prefix='perf-scale-test' \
+                                 -e install_tower=False
+
+
 Looking for Tower deployment on faster disks
 ---------------------------------------------
 
 Run the `ansible-playbook` command with following `EXTRA` cmdline arguments
 
-    ansible-playbook create.yml -e ic_instance=2 \
-                                -e name_prefix='perf-scale-test' \
-                                -e vsi_profile='bx2-4x16' \
-                                -e image_id='r014-02843c52-e12b-4f72-a631-931b4bf6589d'
+    ansible-playbook create.yml -e ibmcloud_vsi_count=2 \
+                                -e ibmcloud_vpc_name_prefix='perf-scale-test'
 
 
-Destroy IBM Cloud Instances
-----------------------------
+Destroy IBM Cloud Instances If Tower is Deployed
+------------------------------------------------
+
+Run the `ansible-playbook` command with following `EXTRA` cmdline arguments
+
+    ansible-playbook cleanup.yml -e ibmcloud_vsi_count=2 \
+                                 -e ibmcloud_vpc_name_prefix='perf-scale-test'
+
+
+Looking for Tower deployment with Isolated Nodes
+------------------------------------------------
+    ansible-playbook create.yml -e install_iso=True
```

Comment:
for the consistency with other examples, should we make it 
        `ansible-playbook create.yml -e ibmcloud_vsi_count=2 \
                                     -e ibmcloud_vpc_name_prefix='perf-scale-test' \
                                     -e install_iso=True`

Reply:
yeah sure thing :+1: will push the changes in a bit

Reply:
https://github.com/jainnikhil30/ibm_cloud_tower_deploy/pull/10/commits/f17e4354e907fdda31f2f0d768319da2c917dc55 adds the above suggested changes